.github/workflow/mk-docs.yml

```diff
@@ -0,0 +1,21 @@
+name: mk-docs
```

Comment:
is any difference between this file and the `build-docs.yml`?

Reply:
If I look at it, yes. This is put together from how its shown on mkdocs. Do you want me to move this into the build-docs.yml?

Reply:
Yes lets move it to `build-docs.yml` and use `requirements-dev` for installation

Reply:
Will do

Reply:
Made the change